src/core/openthread-core-default-config.h

```diff
@@ -753,13 +753,16 @@
 #endif
 
 /**
- * @def OPENTHREAD_CONFIG_ENABLE_BEACON_RSP_IF_JOINABLE
+ * @def OPENTHREAD_CONFIG_ENABLE_BEACON_RSP_WHEN_JOINABLE
  *
- * Define to 1 if you want to enable beacon response for joinable networks.
+ * Define to 1 if you want to enable beacon response device is made joinable.
```

Comment:
suggestion: "Define to 1 to enable IEEE 802.15.4 Beacons when joining is enabled."